soapy_cake.gemspec

```diff
@@ -7,19 +7,20 @@ require 'soapy_cake/version'
 Gem::Specification.new do |spec|
   spec.name          = 'soapy_cake'
   spec.version       = SoapyCake::VERSION
-  spec.authors       = ['ad2games GmbH']
-  spec.email         = ['developers@ad2games.com']
+  spec.required_ruby_version = '~> 2.5.3'
+  spec.authors       = ['Combostrike GmbH']
+  spec.email         = ['developers@combostrike.com']
   spec.summary       = 'Simple client for the CAKE API'
   spec.description   = 'Simple client for the CAKE API (http://getcake.com)'
-  spec.homepage      = 'http://ad2games.com'
+  spec.homepage      = 'http://combostrike.com'
   spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport'
+  spec.add_dependency 'activesupport', '~> 6.1.4'
```

Comment:
How about `~> 6.1`, instead, to make it less restrictive?

Reply:
I intended the restriction. I want everything to be coherent and clear.

Reply:
This is a gem, though, which may or may not be used by others, who may or may not use other versions of `activesupport`.

Reply:
I agree. I think anyway we have to upgrade the projects where this library is being used to at least `6.1`, which wouldn't do much. I will specify the version to be greater or equal to 6.1.